_posts/2024-02-19-jco-1.0.md

```diff
@@ -0,0 +1,301 @@
+---
+title: "Announcing Jco 1.0"
+author: "Yosh Wuyts"
+date: "2024-02-22"
+github_name: "yoshuawuyts"
+excerpt_separator: <!--end_excerpt-->
+---
+
+We’re happy to announce the 1.0 release of [Jco][jco]: a native Javascript
+WebAssembly toolchain and runtime built for WebAssembly Components and [WASI
+0.2](https://bytecodealliance.org/articles/WASI-0.2) [^wasi-one-liner]. Jco can
+natively run Wasm Components inside of Node.js, making it easy to take libraries
+written in different programming languages and execute them using the Node.js
+runtime. And by implementing the entirety of the WASI 0.2 API surface, those
+components can access the network, filesystem, and other system APIs available
+in the Node.js runtime.
+
+[^wasi-one-liner]: WebAssembly Components are an extension to WebAssembly to
+create strongly typed interfaces between different programs. WASI 0.2 is a
+re-work of the "WebAssembly System Interfaces" based on WebAssembly Components.
+The previous version of WASI was not based on components, and as a result was
+rather limited in what it could express.
+
+<!--end_excerpt-->
+
+Our goal is for Jco to be a comprehensive tool for all Component-related
+operations for JavaScript. With Jco 1.0 we're stabilizing a Node.js runtime for
+Wasm Components, as well as a toolchain to take Wasm Components written in other
+languages and import them into JavaScript. From here on out we hope to continue
+stabilizing more features for Jco. Some features are already available
+experimentally, including native support for the browser. As well as native
+support for compiling JavaScript code into WebAssembly.
```

Comment:
Perhaps join these sentences:

```suggestion
stabilizing more features for Jco. Some features are already available
experimentally including native support for the browser as well as native
support for compiling JavaScript code into WebAssembly.
```

Reply:
sure, changed the punctuation here to make it a single sentence. I'm trying to make sure it doesn't turn into a run-on though; so forgive my use of semis.